src/Nethermind/Nethermind.Facade/Simulate/SimulateReadOnlyBlocksProcessingEnv.cs

```diff
@@ -71,7 +71,7 @@ public SimulateReadOnlyBlocksProcessingEnv(
         BlockhashProvider = new SimulateBlockhashProvider(new BlockhashProvider(BlockTree, specProvider, StateProvider, logManager), BlockTree);
         StateProvider = WorldStateManager.GlobalWorldState;
         StateReader = WorldStateManager.GlobalStateReader;
-        CodeInfoRepository = new OverridableCodeInfoRepository(new CodeInfoRepository());
+        CodeInfoRepository = new OverridableCodeInfoRepository(new CodeInfoRepository(1));
```

Comment:
Why are we passing `1` as `chainId` instead of `specProvider.ChainId`?

Reply:
wrong PR